src/main/webapp/components/topbar.jsp

```diff
@@ -0,0 +1,8 @@
+<div id="top-bar">
```

Comment:
Do you want to add a comment (either here or in the readme) on how to use this for future pages?

Reply:
Ack and will include